crates/db_views/src/post_view.rs

```diff
@@ -433,6 +449,9 @@ pub struct PostQuery<'a> {
   pub is_profile_view: bool,
   pub page: Option<i64>,
   pub limit: Option<i64>,
+  // optional when_* parameters are in milliseconds since 00:00:00 UTC on 1 January 1970
```

Comment:
Use three `///` to make it a field comment.

Reply:
ok, changing